README.md

```diff
@@ -56,99 +58,79 @@ $ pod install
 $ open MSALiOS.xcworkspace
 ```
 
-## Step 2: (Optional) 1A: Register your App  
-The app comes pre-configured for testing.  If you would like to register your own app, please follow the steps below.
+## Step 2: (Optional) 
 
-To Register,
-1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account.
+## 2A: Register your App  
 
-2. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations**
+This app comes pre-configured for testing.  If you would like to register your own app, please follow the steps below.
 
-3. You will need to have a native client application registered with Microsoft using the [App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) experience.
+To Register an app:
 
-To create an app,  
-1. Click the **New registration** button on the top left of the page.
-2. On the app registration page,
-    - Name your app
-    - Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts**
-    - Select **Register** to finish.
-3. After the app is created, you'll land on your app management page. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app.bundle.id>://auth`. Replace <app.bundle.id> with the **Bundle Identifier** for your application.
-4. Hit the **Save** button in the top left, to save these updates. 
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account.
+2. In the left-hand navigation pane, select the **Azure Active Directory** blade, and then select **App registrations**.
+3. Click on the **New registration** button at the top left of the page.
+4. On the app registration page,
+   - Name your app
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+   - Click **Register** to finish.
+5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 1C below.
```

Comment:
What is step 1C? Should it be removed from instructions?

Reply:
Fixed

Reply:
Fixed. It should be 2B instead.